source/value-of.d.ts

```diff
@@ -0,0 +1,13 @@
+/**
```

Comment:
If there has been any discussion somewhere about this type, include a link to it. For example, a discussion on the TypeScript issue tracker. - https://github.com/sindresorhus/type-fest/blob/master/.github/contributing.md

Reply:
Fixed!